packages/types/surveys.ts

```diff
@@ -547,4 +547,27 @@ export interface TSurveyQuestionSummary<T> {
   }[];
 }
 
+export const ZSurveyFilterCriteria = z.object({
```

Comment:
Is it possible to eliminate one from `TSurveyFilter` and `TSurveyFilterCriteria`, looks quite similar 🤔

Reply:
Yeah, these two are very similar, but one is being used for client-side filtering types and the other is for server-side filter types. It would be a bit hacky to use one at both places. We can discuss if it needs to be fixed or if we can rename the types to make them more meaningful.